src/index.js

```diff
@@ -1,93 +1,100 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+const fs = require('fs'),
+  http = require('http'),
+  path = require('path'),
+  methods = require('methods'),
+  express = require('express'),
+  bodyParser = require('body-parser'),
+  session = require('express-session'),
+  cors = require('cors'),
+  passport = require('passport'),
+  errorhandler = require('errorhandler'),
+  mongoose = require('mongoose');
+
+const isProduction = process.env.NODE_ENV === 'production';
 
 // Create global app object
 const app = express();
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+app.use(require('morgan')('dev'));
+
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+app.use(require('method-override')());
+
+app.use(express.static(`${__dirname}/public`));
+
+app.get('/', (req, res) => res.status(200).json({
+  status: 200,
+  message: 'Welcome To Bare Foot Nomad',
+}));
 
 app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
+  session({
+    secret: 'authorshaven',
+    cookie: { maxAge: 60000 },
+    resave: false,
+    saveUninitialized: false
+  })
 );
 
 if (!isProduction) {
```

Comment:
Please remove all the require in this code and make it Es6 standard

Reply:
Noted! @Ileriayo

Reply:
The above change has been implemented
Please review